tgit

```diff
@@ -38,7 +38,7 @@ list() {
     do
         cd $i
         if [[ "$(git rev-parse --git-dir)" == ".git" ]]; then
-            status=$(tgit_status $i $1)
+            status=$(tgit_icon_status $i $1)
```

Comment:
I think the script should be called `tgit-status` rather than `tgit_icon_status`.

Let's use underscore just for functions and aliases.

And `-` sign to subscripts.